src/components/ProductPopover.vue

```diff
@@ -76,7 +76,7 @@ export default defineComponent({
         }
         return element;
       });
-      this.store.dispatch('order/updatePurchaseOrderItems', this.purchaseOrders.parsed)
+      this.store.dispatch('order/updatePurchaseOrders', this.purchaseOrders)
```

Comment:
As we are deep cloning the value returned from getter, I think then we don't need to deep clone the original object in here https://github.com/hotwax/import/pull/169/files#diff-474eebde566a51c1cce273a03e682271d4d7d8d674218118ee580f38ecf32d90R68